.github/workflows/pr_validate.yaml

```diff
@@ -91,7 +91,7 @@ jobs:
         continue-on-error: true
         run: |-
           REGEX="\[[Xx]\] Started end-to-end tests on the latest commit\."
-    
+
```

Comment:
can you remove the unnecessary new lines?

Reply:
It's not a new newline, I'm just deleting the whitespace character that is there. Should I remove the newlines too?